### DISCUSSION TITLE
Pagination for index.md possible?

### DISCUSSION THREAD

**Discussion by velasco** [mmistakes/jekyll-theme-basically-basic#130] [Q&A] [answered]
Hi,

I noticed that index.html and index.md are behaving differently. Whiley index.html supports pagination, which is great, Markdown is not fully possible. E.g. heading levels are not working in index.html.

Is it possible to make pagination work in index.markdown?

Regards, Francisco

**Comment by mmistakes** [comment, marked as answer]
It is not. This is a limitation with Jekyll's pagination plugin not the theme.

Ref: https://jekyllrb.com/docs/pagination/
